charging_current_range_test.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+import charging_current_range
+class RangeBucketTest(unittest.TestCase):
+    def test_range_details(self):
+        # self.assertTrue(getRangeDetails([4, 5, 1, 2, 2, 1, 1,2])==8)
+        self.assertTrue(charging_current_range.getRangeDetails([4, 5]) ==[(4 , 5 , 2)])
```

Comment:
Simple readable code 👍  The test cases can be enhanced to get good coverage. For example , (1), (1,1,2,2,3,45) etc.. repeating numbers to be handled too.